docs/telemetry.mdx

```diff
@@ -0,0 +1,114 @@
+- What associated framework versions are being used? (e.g. what `next` version is being used along with `@clerk/nextjs`)
+- Usage of new features
+
+<Callout type="info">
+  **Note:** We regularly audit this list to ensure it is an accurate representation of the data we are collecting. To audit telemetry data sent from our SDKs yourself, you can set a `CLERK_TELEMETRY_DEBUG=1` environment variable in your application. In this mode, telemetry events are only logged to the console and not sent to Clerk.
+</Callout>
+
+An example event looks like this:
+
+```js
+{
+  event: 'COMPONENT_MOUNTED',
+  cv: '4.62.1',
+  sdk: '@clerk/nextjs',
+  sdkv: '4.25.6',
+  pk: 'pk_test_YmFsYW5jZWQtY293YmlyZC0xNi5jbGVyay5hY2NvdW50cy5kDXyk',
+  payload: { component: 'SignIn', appearanceProp: false },
+}
+```
+
+- **event** — A unique identifier for the event type
+- **cv** — Clerk Version. the version of the core Clerk library in use
+- **sdk** — SDK. The Clerk SDK that is being used
+- **sdkv** — SDK Version. The version of the Clerk SDK
+- **pk** — Publishable Key. The instance's publishable key
+- **payload** — Each unique event can provide custom data as part of the payload. As seen above, for the `COMPONENT_MOUNTED` event we track the component name and additional data about prop usage
+
+## What about sensitive data?
+
+We will not collect sensitive data from your application or development environment that is not directly related to your implementation of Clerk's SDKs. Notably, we will not collect: environment variables unrelated to Clerk, any information about your users, file paths, contents of files, logs, git remote information, or raw JavaScript errors.
+
+## How is my data protected?
+
+Clerk takes data privacy and protection seriously. Telemetry data is most useful in aggregate form to gain product insights, and the raw data is only available to a small subset of Clerk employees.
+
+We will never share with or sell telemetry data to third parties. The data is used strictly to help improve the Clerk product.
+
+## How do I opt-out?
+
+For meta-framework SDKs, you can opt-out of telemetry collection by setting the environment variable: `CLERK_TELEMETRY_DISABLED=1`. Note that the variable name might differ between frameworks.
+
+If using `@clerk/clerk-react` directly, or using an SDK that doesn't have environment variable support, you can opt out by passing the follow prop to `<ClerkProvider />`:
+
+```tsx
+<ClerkProvider telemetry={false}>
+```
+
+### Next.js
+
+```tsx
+NEXT_PUBLIC_CLERK_TELEMETRY_DISABLED=1
+```
+
+### Remix
+
+```tsx
+CLERK_TELEMETRY_DISABLED=1
+```
+
+### Gatsby
+
+```tsx
+GATSBY_CLERK_TELEMETRY_DISABLED=1
+```
+
+### React
+
+```tsx
+<ClerkProvider telemetry={false}>
+```
+
+### Vanilla JavaScript
+
+```js
+const clerk = new Clerk(publishableKey)
+c.load({ telemetry: false })
+```
+
+### Chrome extension
+
+```tsx
+<ClerkProvider telemetry={false}>
+```
+
+### Expo
+
+```tsx
+<ClerkProvider telemetry={false}>
+```
```

Comment:
Should this be in a `<CodeBlockTabs />` component?

Reply:
@alexisintech I think for this situation I kinda prefer the non-tabs approach. IMO, it makes it easier to scan and quickly find the section for a specific framework. WDYT?

![Screenshot 2023-11-20 at 1 23 11 PM](https://github.com/clerk/clerk-docs/assets/1289701/3125eb56-e885-430b-84eb-267c7f18d9c1)